packages/js/data/src/setting-options/actions.ts

```diff
@@ -0,0 +1,361 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { DispatchFromMap } from '@automattic/data-stores';
+import type { createRegistry } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { TYPES } from './action-types';
+import type {
+	Setting,
+	SettingValue,
+	SettingsGroup,
+	APIError,
+	BatchSettingsError,
+	SettingsState,
+	SettingUpdate,
+	SettingsUpdateObject,
+} from './types';
+import { NAMESPACE } from '../constants';
+
+type WPDataRegistry = ReturnType< typeof createRegistry >;
+type AllSelectors = typeof import('./selectors');
+type CurriedState< F > = F extends (
```

Comment:
I thiiiiiiiink you can get this from `@wordpress/data`

Reply:
Thanks! Updated in 4711d41d253a6383ce37626d6423f75e1cc7bbc2.